### PR TITLE
Doc/text edits to advanced processing examples

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: examples/01-advanced-postprocessing/Advanced_Far_Field.py [hunk 1]
```diff
@@ -25,41 +28,39 @@
 from pyaedt import Hfss
 
 
-#######################################
-# Importing all the module for postprocessing
-
-
+###############################################################################
+# Import All Module for Postprocessing
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Advanced_Far_Field.py:33]:
```suggestion
# Import All Modules for Post-Processing
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 HFSS-Icepack Coupling Analysis
---------------------------------------------
-This Example shows how to create a full project from scratch in HFSS and Icepak (linked to HFSS). the project creates
-a setup, solves it and create post processing output. It includes a lot of commands to show pyaedt Capabilities
-This Example needs PyVista, numpy and matplotlib,  to be installed on the machine to provide advanced post processing features
-This Examples runs on Windows Only using CPython
-"""
+------------------------------
+This example shows how to create a full project from scratch in HFSS and Icepak (linked to HFSS). 
+The project creates a setup, solves it, and creates postprocessing output. It includes a lot of
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py:5]:
```suggestion
The project creates a setup, solves it, and creates post-processing output. It includes several
```

File: examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
---------------------------------------------
-This Example shows how to create a full project from scratch in HFSS and Icepak (linked to HFSS). the project creates
-a setup, solves it and create post processing output. It includes a lot of commands to show pyaedt Capabilities
-This Example needs PyVista, numpy and matplotlib,  to be installed on the machine to provide advanced post processing features
-This Examples runs on Windows Only using CPython
-"""
+------------------------------
+This example shows how to create a full project from scratch in HFSS and Icepak (linked to HFSS). 
+The project creates a setup, solves it, and creates postprocessing output. It includes a lot of 
+commands to show PyAedt capabilities.
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py:6]:
```suggestion
commands to show PyAEDT's capabilities.
```

File: examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py [hunk 4]
```diff
@@ -142,60 +153,66 @@
...
-
-################################################################
-# ICEPAK Model Creation. After HFSS Setup is ready it will be linked to an icepak project to run a coupled physics analysis
-# ipkapp.copy_solid_bodies_from(aedtapp) will import model from HFSS with all material settings
+################################################################################
+# Create an Icepak Model
+# ~~~~~~~~~~~~~~~~~~~~~~
+# After an HFSS setup is ready, link it to an Icepak project to run a 
+# coupled physics analysis.
+# The command ``ipkapp.copy_solid_bodies_from(aedtapp)`` imports a model
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py:168]:
```suggestion
# The :func:`FieldAnalysisIcepak.copy_solid_bodies_from` imports a model 
```

File: examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py [hunk 5]
```diff
@@ -142,60 +153,66 @@
...
 setup_ipk = ipkapp.create_setup("SetupIPK")
 setup_ipk.props["Convergence Criteria - Max Iterations"] = 3
 setup_ipk.update()
 
-################################################################
-# Mesh Settings
-# After created, a mesh operation is accessible for edit or review of parameters
+################################################################################
+# Edit or Review Mesh Parameters
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py:200]: This is a good way to break up "sections."

File: examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py [hunk 6]
```diff
@@ -236,10 +254,10 @@
 aedtapp.post.plot_field_from_fieldplot(plot1.name, project_path=results_folder, meshplot=False, setup_name=setup_name,
                                              intrinsic_dict=intrinsic, imageformat="jpg", view="iso", off_screen=True)
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Hfss_Icepak_Coupling.py:255]:
```suggestion
                                        intrinsic_dict=intrinsic, imageformat="jpg", view="iso", off_screen=True)
```

File: examples/01-advanced-postprocessing/Maxwell2D_Eddy.py [hunk 7]
```diff
@@ -96,21 +111,26 @@
...
 
 M3D.analyze_nominal()
 
-
-###################################################
-#  get_report_data returns a data class with all data produced from the simulation
+###############################################################################
+# Get Report Data 
+# ~~~~~~~~~~~~~~~
+# The :func:`get_report_data` returns a data class with all data produced from
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Maxwell2D_Eddy.py:124]:
```suggestion
# :func:`PostProcessor.get_report_data` returns a data class with all data produced from 
```

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py [hunk 8]
```diff
@@ -1,51 +1,62 @@
...
 m2d.save_project(os.path.join(project_dir,"M2d.aedt"))
 
-###################################################
-#  create rectangle and duplicate it
+###############################################################################
+# Create a Rectangle and Duplicate It
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This example creates a rectangle and then duplicates it.
 
 rect1 = m2d.modeler.primitives.create_rectangle([0,0,0],[20,10],"Rectangle1", "copper")
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Maxwell2D_Transient.py:39]:
```suggestion
rect1 = m2d.modeler.primitives.create_rectangle([0, 0, 0],[20, 10], "Rectangle1", "copper")
```

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py [hunk 9]
```diff
@@ -1,51 +1,62 @@
...
 
-region = m2d.modeler.primitives.create_region([100,100,100,100,100,100])
+###############################################################################
+# Create a Air Region
+# ~~~~~~~~~~~~~~~~~~~
+# This comman creates an air region.
 
-###################################################
-#  Assign Windings to sheets and balloon to air region
+region = m2d.modeler.primitives.create_region([100,100,100,100,100,100])
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Maxwell2D_Transient.py:48]:
```suggestion
region = m2d.modeler.primitives.create_region([100, 100, 100, 100, 100, 100])
```

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py [hunk 10]
```diff
@@ -1,51 +1,62 @@
...
 
+###############################################################################
+# Assign Windings to Sheets and a Balloon to the Air Region
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Maxwell2D_Transient.py:52]:
```suggestion
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: examples/01-advanced-postprocessing/Readme.txt [hunk 11]
```diff
@@ -1,7 +1,8 @@
-Advanced Post-Processing Examples
-=======================================
-Here are a series of examples using AEDT with ``pyaedt`` that requires advanced packages to be run. The can be run only
-on CPython an requires:
-- numpy
-- matplotlib
-- pyvista
+Advanced Postprocessing Examples
+================================
+This series of examples uses AEDT with PyAEDT. It requires that these advanced packages are running:
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Readme.txt:3]:
```suggestion
This series of examples uses AEDT with PyAEDT. It requires that these advanced packages are installed: 
```

File: examples/01-advanced-postprocessing/Readme.txt [hunk 12]
```diff
@@ -1,7 +1,8 @@
...
-- numpy
-- matplotlib
-- pyvista
+Advanced Postprocessing Examples
+================================
+This series of examples uses AEDT with PyAEDT. It requires that these advanced packages are running: 
+- Matplotlib
+- Numpy
+- PyVista
+
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Readme.txt:7]:
```suggestion

Install these with:

.. code::

   pip install matplotlib pyvista numpy

```

File: examples/01-advanced-postprocessing/Touchstone_Management.py [hunk 13]
```diff
@@ -24,38 +26,42 @@
...
-# Get Curves worst Cases
+###############################################################################
+# Get Curve Worst Cases
+# ~~~~~~~~~~~~~~~~~~~~~
+# These example get the worst cases for the curve.
 
 worst_rl, global_mean = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20,worst_is_higher=True,curve_list=rl_list)
 worst_il, mean2 = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20,worst_is_higher=False,curve_list=il_list)
 worst_fext, mean3 = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20,worst_is_higher=True,curve_list=fext_list)
 worst_next, mean4 = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20,worst_is_higher=True,curve_list=next_list)
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Touchstone_Management.py:59]:
```suggestion
worst_rl, global_mean = get_worst_curve_from_solution_data(data, freq_min=1, freq_max=20, worst_is_higher=True, curve_list=rl_list)
worst_il, mean2 = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20, worst_is_higher=False, curve_list=il_list)
worst_fext, mean3 = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20, worst_is_higher=True, curve_list=fext_list)
worst_next, mean4 = get_worst_curve_from_solution_data(data,freq_min=1, freq_max=20, worst_is_higher=True, curve_list=next_list)
```

File: examples/01-advanced-postprocessing/Touchstone_Management.py [hunk 14]
```diff
@@ -71,9 +77,3 @@
 ax.plot(freq_data, mag_data4,label=worst_next)
 ax.legend(["Worst RL="+worst_rl,"Worst IL="+worst_il,"Worst FEXT="+worst_fext,"Worst NEXT="+worst_next])
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Touchstone_Management.py:78]:
```suggestion
ax.legend(["Worst RL = " + worst_rl, "Worst IL = " + worst_il, "Worst FEXT = " + worst_fext, "Worst NEXT = " + worst_next])
```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

File: examples/01-advanced-postprocessing/Advanced_Far_Field.py [hunk 15]
```diff
@@ -25,41 +28,39 @@
 from pyaedt import Hfss
 
 
-#######################################
-# Importing all the module for postprocessing
-
-
+###############################################################################
+# Import All Module for Postprocessing
```
Comment (PipKat) [inline examples/01-advanced-postprocessing/Advanced_Far_Field.py:32]:
```suggestion
# Import All Modules for Postprocessing
```

File: examples/01-advanced-postprocessing/Advanced_Far_Field.py (hunk 1 above)
Comment (PipKat) [inline examples/01-advanced-postprocessing/Advanced_Far_Field.py:33]:
```suggestion
# ~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Pull request merged**